.github/workflows/ebur128.yml

```
@@ -0,0 +1,115 @@
+name: ebur128
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  rustfmt-clippy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install stable
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        override: true
+        components: clippy, rustfmt
+
+    - name: Run rustfmt
+      uses: actions-rs/cargo@v1
+      with:
+        command: fmt
+        args: -- --check --verbose
+
+    - name: Run clippy
+      uses: actions-rs/clippy-check@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        args: -- -D warnings --verbose -A clippy::wrong-self-convention
```

```suggestion
        args: -- -D warnings -A clippy::wrong-self-convention
```

Same as above :) And probably want to add `--all-features --all-targets` or similar, and ignore (via `#[allow(missing_safety_doc)]`) for the `capi` module (and only that).

It looks also like `clippy::wrong-self-convention` doesn't have to be ignored? I don't have clippy complaining about that anywhere.

---

I hope I've figured out correctly :)

---

Not exactly :) Should be one job/action/task (how is it called?) with clippy for all the targets/features, but you need to add an `#[allow(clippy::missing_safety_doc)]` in `src/lib.rs` before the `mod capi` line.

---

Oh, in that sense, thanks!

---

Done! :)